Update managed-identity.md to include Object ID (#142)
MicrosoftDocs/microsoft-authentication-library-dotnet: msal-dotnet-articles/advanced/managed-identity.md

### User-assigned managed identities

> For user-assigned managed identities, the developer needs to pass the managed identity client id or object id or the full resource identifier string if client ID is not available when creating <xref:Microsoft.Identity.Client.IManagedIdentityApplication>.

Reviewer: Maybe rephrase this a little.
```suggestion
For user-assigned managed identities, the developer needs to pass either of client id, full resource identifier or object id of the managed identity when creating <xref:Microsoft.Identity.Client.IManagedIdentityApplication>.
```
